torch/jit/_recursive.py

```diff
@@ -240,12 +251,12 @@ def infer_type(name, item):
             concrete_type_builder.add_failed_attribute(name, hint)
 
     # Add @property methods as failed attributes, to give a better error message.
-    for name, value in type(nn_module).__dict__.items():
-        if isinstance(value, property):
-            hint = ("\n(This attribute exists on the Python module, but it's an @property "
-                    "method. @property methods are not yet supported in TorchScript. "
-                    "Please file a feature request on Github)")
-            concrete_type_builder.add_failed_attribute(name, hint)
+    # for name, value in type(nn_module).__dict__.items():
+    #     if isinstance(value, property):
+    #         hint = ("\n(This attribute exists on the Python module, but it's an @property "
+    #                 "method. @property methods are not yet supported in TorchScript. "
+    #                 "Please file a feature request on Github)")
+    #         concrete_type_builder.add_failed_attribute(name, hint)
```

Comment:
So I couldn't figure out the right thing to do here. Properties themselves are syntactic sugar for groups of functions that get/set/delete a given attribute (which is usually declared separately in the class body or in `__init__`). Should they be part of the concrete type? If yes, how should they be represented. If not, what happens if two modules have the same underlying data attributes but differing definitions of identically named properties? Are they considered to be the same type?